packages/cipherstash-proxy/src/config/tandem.rs

```diff
@@ -138,6 +138,11 @@ pub struct DevelopmentConfig {
 ///
 /// ENV vars must be prefixed with `CS_`.
 ///
+/// IMPORTANT!
+/// Config needs to be loaded *before* logging can be initalized
+/// Tracing macros will not output and you may be confused.
+/// Use `println!`` and `eprintln!`` instead.
+///
```

Comment:
Nice

Reply:
Yeah, I confused myself for quite a while. 🫠